Signal/src/environment/SignalApp.m

```diff
@@ -220,6 +220,20 @@ - (void)presentConversationForThread:(TSThread *)thread
     });
 }
 
+- (void)withoutAnimationPresentConversationForThread:(TSThread *)thread action:(ConversationViewAction)action
+{
+    OWSAssertIsOnMainThread();
+    DDLogInfo(@"%@ %s", self.logTag, __PRETTY_FUNCTION__);
+
+    ConversationViewController *conversationVC = [ConversationViewController new];
+    [conversationVC configureForThread:thread action:action focusMessageId:nil];
+
+    UIViewController *homeVC = self.homeViewController;
+    homeVC.lastThread = thread;
```

Comment:
Oh huh... so previously we've home view has always had responsibility for presenting its subview and it's `pushTopLevelViewController:...` and `presentViewControllerWithBlock:...` ensure that any existing subviews are cleared.  We've historically had bugs wherein home view ends up with multiple conversations pushed.  I'm wondering if we should:

* move this logic into home view 
* have home view's `withoutAnimationPresentConversationForThread` (or whatever) method also ensure that only one conversation is pushed at a time.

WDYT?